letsencrypt-auto

```diff
@@ -14,7 +14,15 @@ VENV_PATH=${VENV_PATH:-"$XDG_DATA_HOME/$VENV_NAME"}
 VENV_BIN=${VENV_PATH}/bin
 
 if test "`id -u`" -ne "0" ; then
-  SUDO=sudo
+  if type sudo &>/dev/null; then
+    SUDO=sudo
+  else
+    args=("$@")
+    for i in "${!args[@]}"; do
+      args[$i]="'$(printf "%s" "${args[$i]}" | sed -e "s/'/'\"'\"'/g")' "
+    done
+    exec su root -c "$0 ${args[*]}"
```

Comment:
If the script does not run as root, it will require the user enter their password many times.